File: features/steps/ping.js
const {  When, Then } = require('@cucumber/cucumber');
const assert = require('assert');
const app = require('../../src/server');
const supertest = require('supertest');

Then('i have a ok message', async function () {
    assert.equal(response.status,200);
    assert.equal(response.body["message"],"OK");
});
When(/^i enter \/ping$/, async function () {
    response = await supertest(app).get('/ping');
});

File: features/steps/ping.py
from behave import *


@given("la app esta encendida")
def step_impl(context):
    """
    :type context: behave.runner.Context
    """
    pass


@when("si le pego al /ping")
def step_impl(context):
    """
    :type context: behave.runner.Context
    """
    context.response = context.client.get("/ping")


@then('recibo un mensaje "{}"')
def step_impl(context, message_content):
    """
    :param message_content: String
    :type context: behave.runner.Context
    """
    assert context.response.status_code == 200
    assert context.response.json()['message'] == message_content